v2/emailpassword/nextjs/setting-up-frontend.mdx

```diff
@@ -30,19 +34,20 @@ import AppInfoForm from "/src/components/appInfoForm"
 ```tsx title="pages/auth/[[...path]].tsx"
 import React, { useEffect } from 'react'
 import dynamic from 'next/dynamic'
-import SuperTokens from 'supertokens-auth-react'
+import { ^{recipePreBuiltUINameCapitalLetters} } from "supertokens-auth-react/recipe/^{codeImportRecipeName}/prebuiltui";
 import { redirectToAuth } from 'supertokens-auth-react'
+import { canHandleRoute, getRoutingComponent } from 'supertokens-auth-react/ui'
 
-const SuperTokensComponentNoSSR = dynamic<React.ComponentProps<typeof SuperTokens.getRoutingComponent>>(
-  new Promise((res) => res(SuperTokens.getRoutingComponent)),
+const SuperTokensComponentNoSSR = dynamic<{}>(
```

Comment:
Please check if this produces a warning (I recall seeing something like `{}` matches every object)